docs/contributor/rhel4edge_iso.md

```diff
@@ -108,6 +108,7 @@ The artifact of the build is the `_output/image-builder/microshift-installer-${V
 The `kickstart.ks` file is configured to partition the main disk using `Logical Volume Manager` (LVM). Such partitioning is required for the data volume to be utilized by the MicroShift CSI driver and it allows for flexible file system customization if the disk space runs out.
 
 By default, the following partition layout is created and formatted with the `XFS` file system:
+* BIOS boot partition (1MB)
```

Comment:
Can we add a short note on why this is required.

Reply:
@ggiguash Added.